src/ContinuityEditorPopup.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * ContinuityBrowser
+ * Continuity editors
+ */
+
+/*
+   Copyright (C) 1995-2011  Garrick Brian Meeker, Richard Michael Powell
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "ContinuityBrowser.h"
+#include "basic_ui.h"
+#include "calchartapp.h"
+#include "calchartdoc.h"
+#include "cc_command.h"
+#include "cc_continuity.h"
+#include "cc_sheet.h"
+#include "cc_show.h"
+#include "confgr.h"
+
+#include <wx/dcbuffer.h>
+#include <wx/help.h>
+#include <wx/html/helpctrl.h>
+#include <wx/msgdlg.h>
+#include <wx/statline.h>
+
+// ContinuityEditorPopup, for browser style adjustments
+class ContinuityEditorPopup : public wxDialog {
```

Comment:
I think this is a copy-paste error that crept in while refactoring. This class definition appears to be duplicated between the implementation and header files of `ContinuityEditorPopup`.

Reply:
Oops, good catch